spinnaker/spinnaker_system/aws_smoke_test.py

```diff
@@ -236,16 +236,13 @@ def upsert_load_balancer(self, use_vpc):
          aws_module='elb',
          command='describe-load-balancers',
          args=['--load-balancer-names', load_balancer_name])
-     .contains_pred_list([
-         jp.PathContainsPredicate(
-             'LoadBalancerDescriptions/HealthCheck', health_check),
-         jp.PathPredicate(
-             'LoadBalancerDescriptions/AvailabilityZones{0}'.format(
-                 jp.DONT_ENUMERATE_TERMINAL),
-             jp.LIST_SIMILAR(expect_avail_zones)),
-         jp.PathElementsContainPredicate(
-             'LoadBalancerDescriptions/ListenerDescriptions', listener)
-         ])
+     .contains_path_match(
+          'LoadBalancerDescriptions',
+          {'HealthCheck': jp.DICT_MATCHES(
```

Comment:
Sure, but this is an incremental step. I think it is better than using contains_pred_list. My priority was to migrate the registry over into the spinnaker repository and wanted to point in this direction rather than using contains_pred_list so I can get rid of that and use this style of predicate ("name -> predicate" as opposed to "name->operand") to be more consistent with using the more concisely expressive notation.